src/test/java/InvalidLoginTests.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+
+public class InvalidLoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage = new LoginPage(driver);
+        driver.get(url);
+    }
+    @Test
+    public void invalidPasswordEmail(){
+        loginPage.provideEmail("incorrectEmail").providePassword("incorrectPwd").clickSubmit();
+    }
+    @Test
+    public void emptyEmail(){
+        loginPage.provideEmail("").providePassword("incorrectPwd").clickSubmit();
+    }
+    @Test
+    public void emptyEmailPassword(){
+        loginPage.provideEmail("").providePassword("").clickSubmit();
+    }
+}
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**

    Please, add Assert

**One test - One Assert**

For example


![Clean_Test](https://github.com/LiudmilaChernyh/A49-QA-Automation-Framework/assets/104761482/618203bc-5b4c-4d06-811d-3756c5df8f44)

Reply:
So, should I change place of steps here? Or what? I am confused.

Reply:
> So, should I change place of steps here? Or what? I am confused.

Don't worry, just add an Assert

Reply:
I really didn't understand where and how to add assertion. Committed the code for the second time but I still can't run the code. In the downloaded files there is zero information about assertion, or maybe I didn't see it. There were 3 hint pages and I add Home Page to my code as it was advised, not sure why, because I don't have any playlist tests in my work. 
Any other hints? During the live class,  I don't know... I didn't understand the assertion at all.